### PR TITLE
[Feature](Materialized-View) support duplicate base column for diffrent aggregate function

### PR DESCRIPTION
**Pull request by BiteTheDDDDt**
# Proposed changes

1. support duplicate base column for diffrent aggregate function

```sql
create materialized view k12s3m as select k1,sum(k2),max(k2) from d_table group by k1;
```
2. support expression in aggregate function.
```sql
create materialized view k12sa as select k1,sum(abs(k2)) from d_table group by k1;
```
## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
5. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
6. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.26 seconds
 load time: 493 seconds
 storage size: 17122365945 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230111145917_clickbench_pr_77896.html

File: be/src/olap/schema_change.cpp [hunk 1]
```diff
@@ -427,7 +416,7 @@ Status VSchemaChangeDirectly::_inner_process(RowsetReaderSharedPtr rowset_reader
         auto ref_block = std::make_unique<vectorized::Block>(base_tablet_schema->create_block());
 
         rowset_reader->next_block(ref_block.get());
-        if (ref_block->rows() < 1) {
+        if (ref_block->rows() == 0) {
```
Comment (cambyzju) [inline be/src/olap/schema_change.cpp:419]:
```suggestion
        if (ref_block->empty()) {
```

Comment (BiteTheDDDDt) [reply be/src/olap/schema_change.cpp:419]: block has no method named `empty`. XD

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**